nixos/modules/services/web-apps/onlyoffice.nix

```diff
@@ -104,34 +106,53 @@ in
 
         virtualHosts.${cfg.hostname} = {
           locations = {
+            # resources that are generated and thus cannot be taken from the cfg.package yet:
+            "~ ^(\\/[\\d]+\\.[\\d]+\\.[\\d]+[\\.|-][\\w]+)?\\/(sdkjs/common/AllFonts.js)$".extraConfig = ''
+              proxy_pass http://onlyoffice-docservice/$2$3;
+            '';
+            "~ ^(\\/[\\d]+\\.[\\d]+\\.[\\d]+[\\.|-][\\w]+)?\\/(fonts/.*)$".extraConfig = ''
+              proxy_pass http://onlyoffice-docservice/$2$3;
+            '';
             # /etc/nginx/includes/ds-docservice.conf
-            "~ ^(\\/[\\d]+\\.[\\d]+\\.[\\d]+[\\.|-][\\d]+)?\\/(web-apps\\/apps\\/api\\/documents\\/api\\.js)$".extraConfig =
+            #disable caching for api.js
```

Comment:
changes below are the result of checking against `/etc/nginx/includes/ds-docservice.conf` from upstream